swagger.json

```diff
@@ -4508,6 +4624,27 @@
             }
           }
         },
+        "HosCertifiedDayChartResponse": {
+            "type": "object",
+            "properties": {
+                "dayStartMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unix epoch time (in ms) of time of when this daily log starts",
+                    "example": 1462881998034
+                },
+                "shippingDocNumbers": {
+                    "type": "string",
+                    "description": "Customer Shipping Document Numbers, potentially multiple numbers delimited by a comma or space",
```

Comment:
nit: is `Customer Shipping Document Numbers` a proper noun? If not let's make it `Customer shipping document numbers`.

Reply:
done